src/entities/expenses/ui/ExpensesTotalMoneyAmount/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState, type FC } from 'react';
+import { observer } from 'mobx-react';
+import { autorun } from 'mobx';
+import { useStore } from '@shared/lib/useStore';
+
+export const ExpensesTotalMoneyAmount: FC = observer(() => {
+  const { expensesStore } = useStore();
+  const [amount, setAmount] = useState(0);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    if (expensesStore.getAmount() !== 0) {
+      setLoading(false);
+    }
+    const disposer = autorun(() => {
+      const updatedAmount = expensesStore.getAmount();
+      setAmount(updatedAmount);
+      if (updatedAmount !== 0) {
+        setLoading(false);
+      }
+    });
+    return () => {
+      disposer();
+    };
+  }, [expensesStore]);
```

Comment:
1. U can separate loading in expenses store, then u should do autorunning
2. It's business logic, should separate it to `models`